packages/snaps-sdk/src/jsx/components/form/Field.ts

```diff
@@ -19,6 +20,8 @@ export type FieldProps = {
   error?: string | undefined;
   children:
     | [InputElement, ButtonElement]
+    | [BoxElement, InputElement]
+    | [InputElement, BoxElement]
```

Comment:
Yea, actually I was just testing this. Pushed now. 

So, I allowed Box on both sides now. Button can be also Boxed on the left or on the right side. I avoided adding all the combinations with Button, since it works with Box anyway. Let me know if you have different thoughts on that.

Reply:
![Screenshot 2024-09-04 at 17 25 51](https://github.com/user-attachments/assets/1eea93d0-34a5-4e10-aa0c-96c6370f3655)
Some of the examples I'm testing with right now while implementing extension support.